sdk/schemaregistry/azure-data-schemaregistry/src/test/java/com/azure/data/schemaregistry/SchemaRegistryAsyncClientPlaybackTests.java

```diff
@@ -110,7 +110,7 @@ public void getSchemaByIdFromPortal() {
 
     /**
      * Verifies that the new serializer works with 1.0.0 schema registry client.
-     * https://search.maven.org/artifact/com.azure/azure-data-schemaregistry/1.0.0/
+     * https://contral.sonatype.com/search?namespace=com.azure&name=azure-data-schemaregistry
```

Comment:
This link should also include the version as this is test is specific for a given version.